pkg/storaged/btrfs/subvolume.jsx

```diff
@@ -44,19 +47,137 @@ function subvolume_mount(volume, subvol, forced_options) {
     mounting_dialog(client, block, "mount", forced_options, subvol);
 }
 
+function get_mount_point_in_parent(volume, subvol) {
+    const block = client.blocks[volume.path];
+    const subvols = client.uuids_btrfs_subvols[volume.data.uuid];
+    if (!subvols)
+        return null;
+
+    for (const p of subvols) {
+        const has_parent_subvol = subvol.pathname.substring(0, p.pathname.length) == p.pathname &&
```

Comment:
`has_parent_subvol` is never true when `p` is the root subvolume (p.pathname == "/").  I think this needs to be

```
        const has_parent_subvol = (p.pathname == "/" && subvol.pathname  !== "/") ||
                                  (subvol.pathname.substring(0, p.pathname.length) == p.pathname &&
                                   subvol.pathname[p.pathname.length] == "/"));
```